dev/WebView2/WebView2.cpp

```diff
@@ -1505,6 +1505,38 @@ void WebView2::TryCompleteInitialization()
             [this](auto&&, auto&&) { UpdateDefaultBackgroundColor(); });
     }
 
+    AddChildPanel();
+    CreateAndSetVisual();
+
```

Comment:
This should be under #ifdef WINUI2 I think?

Reply:
We have plenty of code in WinUI 2 that's not in WinUI 3 that we don't #ifdef. If anything, we should just remove the WinUI 3 bits we left in. If we want to go the other way and start adding ifdefs for everything, I think that should be its own effort.